### ISSUE TITLE
Add access-key's last_used_date to user's filter table

### ISSUE THREAD

**Issue by chris-redekop** [chef/inspec-aws#72]
For recommendation 1.4.

**Comment by clintoncwolfe** [comment]
Recommendation 1.4 is about rotation, i.e. age (today minus creation date < 90): 
```
Ensure access keys are rotated every 90 days or less (Scored) 
```
So I think access key creation date #64 is sufficient?

**Comment by chris-redekop** [comment]
I agree that we want #64.  I think we also want #72 and #75, for one detail from Recommendation 1.4:

> Ensure all active keys have been used since last rotation. Keys not in-use since last rotation should be disabled/deleted.

In #18, Christoph mentions something like the following for this test, but I'm not sure how to express the final `create_date < last_used` check.

```
control 'iam-test-01' do
   title 'test access keys'
  aws_iam_users.where { access_key.active? == true }.access_keys { |access_key|
    describe.one do
      # Either the key has been created in the last hour OR
      describe access_key do
        its { create_date } { should be > Time.now - 3600 }
      end
      # the key has been used since it was created.  How to express this?
      describe access_key do
        its { last_used_date } { should be > create_date }
      end
    end
  }
end
```

**Comment by clintoncwolfe** [comment]
Taking off my tunnel-vision glasses, I see that recommendation 1.3 certainly requires last_used_date (or time, or whatever we get).  So, I withdraw my objection :-)

For testing whether a key has (ever) been used, I'd imagine last_used would be nil.  AWS CLI says it will be null if never used - http://docs.aws.amazon.com/cli/latest/reference/iam/get-access-key-last-used.html . 

```
describe access_key do
    its( 'last_used_date') { should_not be nil }
end
```

**Comment by chris-redekop** [comment]
Closed by #112.
